statemachine/factory.py

```diff
@@ -179,7 +179,7 @@ def add_inherited(cls, bases):
                 cls.add_event(event=Event(id=event.id, name=event.name))
 
     def add_from_attributes(cls, attrs):  # noqa: C901
-        for key, value in sorted(attrs.items(), key=lambda pair: pair[0]):
+        for key, value in attrs.items():
```

Comment:
The library is pre-python3.7 where the dictionary order was not guaranteed. This order hack is not required since 3.7.